model/src/test/java/com/javahelp/model/usertest/ClientUserInfoTest.java

```diff
@@ -1,4 +1,4 @@
-package com.javahelp.model.userTest;
+package com.javahelp.model.usertest;
```

Comment:
package should just be `com.javahelp.model.user` on these. It is conventional to keep tests in same package as the classes they pertain to.